Script/PRONTO.py

```diff
@@ -231,7 +232,123 @@ def clear_blank_line(file_in,file_out):
 	os.remove(file_in)
 
 
-def get_patient_info_from_MTF(ipd_material_file,ipd_no,DNA_sampleID,RNA_sampleID):
+def get_patient_info_from_MTF_2023(ipd_material_file,ipd_no,DNA_sampleID,RNA_sampleID):
+	import xlrd
```

Comment:
Currently only OUS uses MTF file to import the clinical data so this module is imported when this function is needed.

Reply:
Ah ok, I understand your reasoning. I was not sure what the recommendation is either and it seems, imports should be on top of the file. This also has to do with import times and it might potentially take more time to run a function if the import was not done in the beginning of the file. I also found [this](https://realpython.com/absolute-vs-relative-python-imports/#styling-of-import-statements). I also thought that these functions specific for OUS could go into their own module which would make the code more readable as well. For individual modules, you can import the packages on top of the file as well. Maybe a little refactor can easily solve this.

Reply:
Emm, I see. I could create an issue to move the import module to the top instead of functions. The separate module for OUS could be planed later.